scope.go

```diff
@@ -55,27 +59,33 @@ var (
 )
 
 type scope struct {
-	separator      string
-	prefix         string
-	tags           map[string]string
-	reporter       StatsReporter
-	cachedReporter CachedStatsReporter
-	baseReporter   BaseStatsReporter
-	defaultBuckets Buckets
-	sanitizer      Sanitizer
-
-	registry *scopeRegistry
-	status   scopeStatus
+	separator           string
+	prefix              string
+	tags                map[string]string
+	reporter            StatsReporter
+	cachedReporter      CachedStatsReporter
+	baseReporter        BaseStatsReporter
+	defaultBuckets      Buckets
+	sanitizer           Sanitizer
+	expirePeriodSeconds int64
+
+	registry    *scopeRegistry
+	status      scopeStatus
+	createdUnix int64
```

Comment:
is there an efficiency win from storing unix seconds rather than storing `*time.Time` and comparing using `.After`?

Reply:
Yeah about 20% difference in performance. Figured the logic wasn't that hard to read so worth the perf win.